frontend/device/kindle/device.lua

```diff
@@ -936,17 +943,14 @@ function KindleOasis:init()
                     rotation_mode = self.screen.DEVICE_ROTATED_COUNTER_CLOCKWISE
                 end
             end
-
-            if rotation_mode > 0 then
-                self.screen.native_rotation_mode = rotation_mode
-                self.screen.cur_rotation_mode = rotation_mode
-            end
-
+            logger.dbg("rotation_mode = ", rotation_mode)
```

Comment:
That one is probably unnecessary though, the conversion is pretty straightforward ;).

Reply:
OK!